app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/SettingsDownloadsLimitScreen.kt

```diff
@@ -72,7 +83,11 @@ enum class DownloadsLimit(
 private fun SettingsThemeScreenPreview() {
     SwissTransferTheme {
         Surface {
-            SettingsDownloadsLimitScreen {}
+            SettingsDownloadsLimitScreen(
+                downloadLimit = DownloadLimit.TWOHUNDREDFIFTY,
```

Comment:
Actually, this is in `multiplatform_swisstransfer`.
It needs to be fixed there.

Reply:
Fixed here : https://github.com/Infomaniak/multiplatform-SwissTransfer/pull/23